src/Cpp/1-getting-started/1-1-3-HelloTriangle/HelloTriangle.hpp

```diff
@@ -28,14 +28,14 @@ class HelloTriangleApplication final : public Application
     bool CreateSwapchainResources();
     void DestroySwapchainResources();
 
-bool CompileShader(
-    const std::wstring_view fileName,
-    const std::string_view entryPoint,
-    const std::string_view profile,
-    ComPtr<ID3D10Blob>& shaderBlob) const;
+    bool CompileShader(
```

Comment:
These are tabs tabs? or smart tabs? :P

Reply:
they are spaces, I thought this wasn't aligned with the other declarations?

Reply:
Oki doki, its not clear in this shit review screen if its spaces or tabs :) But i noticed the missing indent too :)